pkgs/amtool.nix

```
@@ -1,21 +1,51 @@
-{ stdenv
-, lib
-, prometheus-alertmanager
+{ lib
+, go
+, buildGoModule
+, fetchFromGitHub
+, installShellFiles
+, source
 }:
 
-stdenv.mkDerivation {
+buildGoModule rec {
   pname = "amtool";
-  version = prometheus-alertmanager.version;
+  version = "0.24.0";
+  rev = "v${version}";
 
-  unpackPhase = ":";
-  installPhase = ''
-    install -m755 -D ${prometheus-alertmanager}/bin/amtool $out/bin/amtool
+  src = fetchFromGitHub {
+    inherit rev;
+    owner = source.owner;
+    repo = source.repo;
+    sha256 = source.sha256;
+  };
+
+  vendorSha256 = "sha256-ePb9qdCTEHHIV6JlbrBlaZjD5APwQuoGloO88W5S7Oc=";
```

```suggestion
  vendorSha256 = source.vendorSha256;
```
And add manually the vendorSha256 in the `source.json`. This is for the autoupdate script.

---

Fixed